app/services/test_service/update.rb

```diff
@@ -0,0 +1,33 @@
+# PATH: /app/services/test_service/update.rb
+# rubocop:disable Style/ClassAndModuleChildren
+class TestService::Update
+  attr_accessor :id, :name, :status
+  def initialize(id, name, status)
+    @id = id
+    @name = name
+    @status = status
```

Comment:
The `TestService::Update` service class is using manual validation which can be error-prone and hard to maintain. Consider using ActiveModel validations for better maintainability.